components/google_drive/actions/add-file-sharing-preference/add-file-sharing-preference.js

```diff
@@ -0,0 +1,97 @@
+const googleDrive = require("../../google_drive.app");
+
+module.exports = {
+  key: "google_drive-add-file-sharing-preference",
+  name: "Add File Sharing Preference",
+  description:
+    "Add a sharing permission to the sharing preferences of a file and provide a sharing URL",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleDrive,
+    /* eslint-disable pipedream/default-value-required-for-optional-props */
```

Comment:
I used an ESLint configuration comment to disable the `pipedream/default-value-required-for-optional-prop` (and silence the warning). `/* eslint-disable */` disables the rule for the entire file. Should we use `/* eslint-disable-next-line */` (which disables the rule for the next line) instead before each optional prop without a default value?